one_fm/one_fm/doctype/erf/erf.json

```diff
@@ -867,19 +865,11 @@
    "label": "Default Shift",
    "mandatory_depends_on": "eval:doc.shift_working==0",
    "options": "Shift Type"
-  },
-  {
-   "depends_on": "eval: doc.hiring_method == 'Bulk Recruitment'",
-   "fieldname": "interview_round",
-   "fieldtype": "Link",
-   "label": "Interview Round",
-   "mandatory_depends_on": "eval: doc.hiring_method == 'Bulk Recruitment'",
-   "options": "Interview Round"
```

Comment:
It should be there in the ERF

Reply:
This section has been added back to the json file